dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -58,6 +58,9 @@ endpoint.virtualHost.desc=ID of a configured virtual host. All endpoints are ass
 endpoint.protocolVersion=Protocol version
 endpoint.protocolVersion.desc=When Servlet 4.0 API is enabled as a feature, set this attribute to http/1.1 to disable HTTP/2 processing for the ports that were defined for the httpEndpoint element. When Servlet 3.1 API is enabled as a feature, set this attribute to http/2 to enable HTTP/2 processing for the ports that are defined for the httpEndpoint element.
 
+endpoint.transportVersion=Transport version
+endpoint.transportVersion.desc=When left undefined, or when this attribute is set to Legacy, the IBM Channel Framework Transport will used. If this attribute is set to Netty, the Netty implementation will be used to start the endpoint.
```

Comment:
suggested edit:

If this attribute is either undefined or set to Legacy, the IBM Channel Framework Transport is used start the endpoint. If this attribute is set to Netty, the Netty implementation is used to start the endpoint.

Reply:
Sorry this isnt meant to be a part of this PR, I'll remove it